### DISCUSSION TITLE
Why does cursor briefly jump to neotree after command get executed?

### DISCUSSION THREAD

**Discussion by marcinjahn** [neovide/neovide#2727] [Q&A]
Have a look at the following clip:

https://github.com/user-attachments/assets/f9322c26-f7a3-4da9-9380-306603efb1ff

I have a typical layout: a neotree and an editor. I'm invoking a command to save a file. Right after that gets executed, the cursor:

1. Jumps to neotree
2. Jumps to the editor

Why point 1 even occurs? Is it a neovide thing? Is it how neovim works? Or some plugin?

**Comment by fredizzimo** [comment]
This is probably the same issue as this
* https://github.com/neovide/neovide/issues/2735

But the puzzling thing is that your report appears to be before noice 4.5.0 was released.
